dockerfiles/che/entrypoint.sh

```diff
@@ -262,8 +262,6 @@ init() {
   sed -i "/che.workspace.storage=/c\che.workspace.storage=/data/workspaces" $CHE_LOCAL_CONF_DIR/che.properties
   sed -i "/che.database=/c\che.database=/data/storage" $CHE_LOCAL_CONF_DIR/che.properties
   sed -i "/che.template.storage=/c\che.template.storage=/data/templates" $CHE_LOCAL_CONF_DIR/che.properties
-  sed -i "/che.stacks.storage=/c\che.stacks.storage=/data/stacks/stacks.json" $CHE_LOCAL_CONF_DIR/che.properties
-  sed -i "/che.stacks.images=/c\che.stacks.images=/data/stacks/images" $CHE_LOCAL_CONF_DIR/che.properties
```

Comment:
yes it's not docker images.
But how is backward compliance for users that were using the stacks.json with custom images for their stacks. It seems now this is no longer possible.

Also the description of the pull request is not describing the path or the property name to define the path to the custom stacks.json to load